src/components/pages/users.js

```diff
@@ -55,7 +54,7 @@ function createUserSubsets(users, storagePlans) {
       internalUsers.push(enrichedUserObject);
     }
 
-    if (user.storage?.plan || user.storage?.usedBytes) {
+    if (user.storage.plan || user.storage.usedBytes) {
```

Comment:
Actually, it also should be shown if there are (for whatever reason) reminders:

`if (user.storage.plan || user.storage.usedBytes || user.storage.reminders.length) {`

Reply:
Good idea, done!